src/index.ts

```diff
@@ -673,6 +682,98 @@ export class ERC725 {
   encodeKeyName(keyName: string, dynamicKeyParts?: DynamicKeyParts): string {
     return encodeKeyName(keyName, dynamicKeyParts);
   }
-}
 
+  baseConfig = { ipfsGateway: IPFS_GATEWAY_BASE_URL };
```

Comment:
Can you not use the ipfs config of the ERC725 class rather than recreate it here?